Button/button.css

```diff
@@ -0,0 +1,108 @@
+.btn {
+    padding: 0.5em 1em 0.5em 1em;
+    
+}
+
+.btn-round-edges {
+    border-radius: 1em;
+    padding: 0.5em 1.5em 0.5em 1.5em;
```

Comment:
instead of using this  padding: 0.5em 1.5em 0.5em 1.5em;

you can use below it will work the same.
 padding: 0.5em 1.5em;
top and bottom paddings are 0.5em
right and left paddings are 1em